src/Snackbar/Snackbar.js

```diff
@@ -169,7 +170,7 @@ type Props = DefaultProps & {
    * @param {event} event The event that triggered the close request
    * @param {string} reason Can be:`"timeout"` (`autoHideDuration` expired) or: `"clickaway"`
    */
-  onRequestClose?: (event: ?SyntheticUIEvent, reason: string) => void,
+  onRequestClose?: (event: ?Event, reason: string) => void,
```

Comment:
`declare class SyntheticUIEvent extends SyntheticEvent`, only small difference, possibly unnecessary but if it's either, I would just use the more base class.